resume.md

```diff
@@ -1,49 +1,43 @@
-# FIRST LAST NAME
+# MAANAV KHAITAN
 
-(123) 456 - 7890 | email@gmail.com
+(510) XXX - XXXX | maanav@berkeley.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/MaanavKhaitan | linkedin.com/in/maanav-khaitan
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**UC Berkeley - Management, Entrepreneurship, & Technology Program** | Dual Degree: B.S. Electrical Engineering & Computer Science + B.S. Business Administration. *May 2024*
+* **Coursework**: Data Structures, Structure and Interpretation of Computer Programs, Designing Information Devices and Systems I & II
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Languages**: Python, Java, JavaScript, Swift, SQL, R
+* **Libraries/Frameworks**: React, Pandas, NumPy, MongoDB, Next.js, jQuery, Flask, Scikit-Learn
+* **Version Control**: Git, GithubDesktop
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Product Engineer Intern**: Edith Labs | Cupertino, California | *September 2020 - Present*
+* Developed full-stack web app to connect students and mentors, in JavaScript, React, Next.js, HTML, CSS, and MongoDB.
```

Comment:
Be specific - what was the point of these web apps?

"I did X for Y to accomplish Z" what did this accomplish?